iron-component-page.html

```diff
@@ -609,6 +612,19 @@
           if (!this._findDescriptor(this.active)) {
             this.active = this._getDefaultActive();
           }
+
+          // Scroll to deep link hash.
+          // TODO: support scrolling to hidden API properties/methods.
+          // TODO: deep link into non-default "active" view in dropdown.
+          this.async(function() {
+            var hash = window.location.hash;
+            if (hash) {
+              var goToElement = this.$.view.querySelector(hash);
+              if (goToElement) {
+                goToElement.scrollIntoView();
```

Comment:
Yea. The content needs to be stamped/in the dom before you can scroll to the element. If you load a deep link directly, the browser will try to go to it right away, but it's not stamped yet.